run/types/DeviceWrapper.ts

```diff
@@ -973,13 +993,13 @@ export class DeviceWrapper {
     await this.waitForTextElementToBePresent({
       strategy: 'accessibility id',
       selector: 'Conversation list item',
-      text: receiver.userName,
+      text: String(receiver.userName),
```

Comment:
is this needed?

Reply:
This seems to have been introduced when i changed over USERNAME to be an ENUM, not a type. 
I've removed it and there doesn't seem to be any issues